package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "KytraMintBean0404",
+  "version": "1.14.0",
+  "description": "MintBean0404",
+  "author": "Kytra",
+  "license": "",
+  "main": "../MintBean04-04/server/server.js",
```

Comment:
why refer to a folder outside the project? Usually a bad idea in package.json :-)